packages/world/src/modules/init/functionSignatures.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.24;
+
+/**
+ * @dev Function signatures for access management system
+ */
+function getFunctionSignaturesAccessManagement() pure returns (string[4] memory) {
+  return [
+    // --- AccessManagementSystem ---
+    "grantAccess(bytes32,address)",
+    "revokeAccess(bytes32,address)",
+    "transferOwnership(bytes32,address)",
+    "renounceOwnership(bytes32)"
+  ];
```

Comment:
WHAT YOU CAN DO THIS

Reply:
We already were! only works for static arrays